src/brod_group_subscriber.erl

```diff
@@ -93,43 +96,46 @@
 %% default value -1 (latest) is used.
 %
 % commented out as it's an optional callback
-%-callback get_committed_offsets(group_id(), [{topic(), partition()}],
-%                                cb_state()) ->
-%            {ok, [{{topic(), partition()}, offset()}], cb_state()}.
-
-
+%-callback get_committed_offsets(brod:group_id(),
```

Comment:
We have `-optional_callbacks` attribute since OTP18 but I guess it's a bit too early to deprecate 17. Just a note.